### PR TITLE
Removed youtube-dl refs and links from CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by makeworld-the-better-one**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

This PR only changes documentation and not code. I have improved the CONTRIBUTING.md file to no longer reference or link to youtube-dl in any way. This makes for a less confusing read, and helps yt-dlp stand on its own as a project. It also prevents accidentally forking youtube-dl instead yt-dlp, which happened to me 😅

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,23 +28,23 @@ So please elaborate on what feature you are requesting, or what bug you want to
...
 
 **Site support requests must contain an example URL**. An example URL is a URL you might want to download, like `https://www.youtube.com/watch?v=BaW_jenozKc`. There should be an obvious video present. Except under very special circumstances, the main page of a video service (e.g. `https://www.youtube.com/`) is *not* an example URL.
 
 ###  Are you using the latest version?
 
-Before reporting any issue, type `youtube-dl -U`. This should report that you're up-to-date. About 20% of the reports we receive are already fixed, but people are using outdated versions. This goes for feature requests as well.
+Before reporting any issue, type `yt-dlp -U`. This should report that you're up-to-date. About 20% of the reports we receive are already fixed, but people are using outdated versions. This goes for feature requests as well.
```
Comment (Ashish0804) [inline CONTRIBUTING.md:39]: IMO this 20% thingy doesn't apply anymore so better to remove it altogether.
```suggestion
Before reporting any issue, type `yt-dlp -U`. This should report that you're up-to-date. This goes for feature requests as well.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -30,23 +28,23 @@ So please elaborate on what feature you are requesting, or what bug you want to
...
 
 ###  Is the issue already documented?
 
-Make sure that someone has not already opened the issue you're trying to open. Search at the top of the window or browse the [GitHub Issues](https://github.com/ytdl-org/youtube-dl/search?type=Issues) of this repository. If there is an issue, feel free to write something along the lines of "This affects me as well, with version 2015.01.01. Here is some more information on the issue: ...". While some issues may be old, a new post into them often spurs rapid activity.
+Make sure that someone has not already opened the issue you're trying to open. Search at the top of the window or browse the [GitHub Issues](https://github.com/yt-dlp/yt-dlp/search?type=Issues) of this repository. If there is an issue, feel free to write something along the lines of "This affects me as well, with version 2015.01.01. Here is some more information on the issue: ...". While some issues may be old, a new post into them often spurs rapid activity.
```
Comment (Ashish0804) [inline CONTRIBUTING.md:43]: I know this is an example but maybe update the version just this time.
```suggestion
Make sure that someone has not already opened the issue you're trying to open. Search at the top of the window or browse the [GitHub Issues](https://github.com/yt-dlp/yt-dlp/search?type=Issues) of this repository. If there is an issue, feel free to write something along the lines of "This affects me as well, with version 2021.09.02. Here is some more information on the issue: ...". While some issues may be old, a new post into them often spurs rapid activity.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -58,23 +56,23 @@ We are then presented with a very complicated request when the original problem
...
+It may sound strange, but some bug reports we receive are completely unrelated to yt-dlp and relate to a different, or even the reporter's own, application. Please make sure that you are actually using yt-dlp. If you are using a UI for yt-dlp, report the bug to the maintainer of the actual application providing the UI. On the other hand, if your UI for yt-dlp fails in some way you believe is related to yt-dlp, by all means, go ahead and report the bug.
 
 # DEVELOPER INSTRUCTIONS
 
-Most users do not need to build youtube-dl and can [download the builds](https://ytdl-org.github.io/youtube-dl/download.html) or get them from their distribution.
+Most users do not need to build yt-dlp and can download the builds or get them from their distribution.
```
Comment (Ashish0804) [inline CONTRIBUTING.md:71]: You can link the GH releases page here

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -95,21 +93,21 @@ If you want to create a build of youtube-dl yourself, you'll need
 
 ### Adding support for a new site
 
-If you want to add support for a new site, first of all **make sure** this site is **not dedicated to [copyright infringement](README.md#can-you-add-support-for-this-anime-video-site-or-site-which-shows-current-movies-for-free)**. youtube-dl does **not support** such sites thus pull requests adding support for them **will be rejected**.
+If you want to add support for a new site, first of all **make sure** this site is **not dedicated to copyright infringement**. yt-dlp does **not support** such sites thus pull requests adding support for them **will be rejected**.
```
Comment (Ashish0804) [inline CONTRIBUTING.md:96]: It wouldn't be a bad idea to port this `can-you-add-support-for-this-anime-video-site` aswell

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -148,45 +146,45 @@ After you have ensured this site is distributing its content legally, you can fo
...
-7. Have a look at [`youtube_dl/extractor/common.py`](https://github.com/ytdl-org/youtube-dl/blob/master/youtube_dl/extractor/common.py) for possible helper methods and a [detailed description of what your extractor should and may return](https://github.com/ytdl-org/youtube-dl/blob/7f41a598b3fba1bcab2817de64a08941200aa3c8/youtube_dl/extractor/common.py#L94-L303). Add tests and code for as many as you want.
-8. Make sure your code follows [youtube-dl coding conventions](#youtube-dl-coding-conventions) and check the code with [flake8](https://flake8.pycqa.org/en/latest/index.html#quickstart):
+7. Have a look at [`yt_dlp/extractor/common.py`](https://github.com/yt-dlp/yt-dlp/blob/master/yt_dlp/extractor/common.py) for possible helper methods and a [detailed description of what your extractor should and may return](https://github.com/yt-dlp/yt-dlp/blob/0001fcb586c3ab297cd48c77ddd6f5d40546dac4/yt_dlp/extractor/common.py#L105-L366). Add tests and code for as many as you want.
+8. Make sure your code follows [yt-dlp coding conventions](#yt-dlp-coding-conventions) and check the code with [flake8](https://flake8.pycqa.org/en/latest/index.html#quickstart):
```
Comment (Ashish0804) [inline CONTRIBUTING.md:155]: we donot have `#yt-dlp-coding-conventions`
Maybe u can port them aswell

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -148,45 +146,45 @@ After you have ensured this site is distributing its content legally, you can fo
...
 
-        $ flake8 youtube_dl/extractor/yourextractor.py
+        $ flake8 yt_dlp/extractor/yourextractor.py
 
-9. Make sure your code works under all [Python](https://www.python.org/) versions claimed supported by youtube-dl, namely 2.6, 2.7, and 3.2+.
+9. Make sure your code works under all [Python](https://www.python.org/) versions claimed supported by yt-dlp, namely 2.6, 2.7, and 3.2+.
```
Comment (Ashish0804) [inline CONTRIBUTING.md:159]:
```suggestion
9. Make sure your code works under all [Python](https://www.python.org/) versions claimed supported by yt-dlp, namely 3.6+
```
Tho the whole line should be reworded imo

**Comment by Ashish0804** [comment]
Also just realized this exists https://github.com/yt-dlp/yt-dlp/pull/528

**Comment by pukkandan** [comment]
I'll clean up and merge #528 instead. But it's gonna take a bit of time

**Pull request closed without merging**